app/build.gradle

```diff
@@ -12,12 +12,18 @@ android {
         versionName versionName
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
     }
+    lintOptions {
+        abortOnError false
+    }
     buildTypes {
         release {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+    packagingOptions {
```

Comment:
Why did you added this? I thought it fixed the issue https://github.com/algolia/instantsearch-android-insights/commit/c774fa6efdc0eed187ab477fabcd81550fac11fa

Reply:
Somehow it prevented me from compiling the sample app, but now it looks like it is working again .. Not sure why.